locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -4481,6 +4481,10 @@ msgstr ""
 msgid "Are you sure you want to delete this request?"
 msgstr ""
 
+#: corehq/apps/users/static/users/js/web_users.js
+msgid "s email server"
+msgstr ""
```

Comment:
This is driving me nuts. I have no idea where this is coming from. On master if you run

`./scripts/make-translations.sh`

it creates no diff (the files are already up to date).

If you run `tx pull -f; ./scripts/make-translations.sh` there are a bunch of other substantive translation changes (as you'd expect) that you see in this PR, but **not this one**. Which is very confusing. I'll keep on it

Reply:
Update, I might have confused myself and run `tx pull` instead of `tx pull -f`. Right after running `tx pull -f` now, I see this "s email server" string added. I can see that it exists in transifex (in the online portal), so it seems like it's getting pulled from there.

I'm still a little confused how something that's in transifex ever gets deleted when we remove it on the code side, since the code by itself is not generating this translation. 🤔